### PR TITLE
vita3K: fix format.

### PR DESCRIPTION
**Pull request by Zangetsu38**

### CODE REVIEW

**Review by bookmist — APPROVED** [review]

File: vita3k/util/include/util/vector_math.h [hunk 1]
```diff
@@ -238,35 +241,35 @@ inline SceFloat Vec3<SceFloat>::Normalize() {
...
-    Lerp(const X &begin, const X &end, const SceFloat t) {
+Lerp(const X &begin, const X &end, const SceFloat t) {
     return begin * (1.f - t) + end * t;
 }
 
 // linear interpolation via int: 0=begin, base=end
 template <typename X, int base>
-[[nodiscard]] constexpr decltype((X{} * int{} + X{} * int{}) / base) LerpInt(const X& begin,
-                                                                             const X& end,
-                                                                             const int t) {
```
Comment (IllusionMan1212) [inline vita3k/util/include/util/vector_math.h:262]: Not sure if the others agree with these changes since it's easier to read before

Comment (Zangetsu38) [reply vita3k/util/include/util/vector_math.h:262]: macdu have said is fine in my other pr

**Comment by nn9dev** [comment]
Why was this merged with the ctrl changes?

**Comment by Zangetsu38** [comment]
> Why was this merged with the ctrl changes?

you wrong look
is just format fix, your code format is full broken

**Comment by nn9dev** [comment]
I think "full broken" is a little harsh, but I see, I was mistaken

**Comment by Zangetsu38** [comment]
Sorry, it was to say that the format was invalid, everything was shifted and not in the right place

**Pull request merged**